app/log/config.go

```diff
@@ -229,7 +235,7 @@ func newStructuredLogger(format string, level zapcore.Level, ws zapcore.WriteSyn
 	return zap.New(
 		zapcore.NewCore(structured, ws, zap.NewAtomicLevelAt(level)),
 		zap.WithCaller(true),
-		zap.AddCallerSkip(1),
```

Comment:
had to make this dynamic based on whether logger are wrapper by multilogger or not.